src/components/AutoComplete/AutoComplete.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <Combobox as="div" v-model="displayValue" :disabled="disabled">
```

Comment:
minor: could you just add the semantic component class here
```suggestion
  <Combobox as="div" class="concrete__autocomplete" v-model="displayValue" :disabled="disabled">
```